build.ps1

```diff
@@ -0,0 +1,108 @@
+[CmdletBinding(DefaultParameterSetName='RegularBuild')]
+param (
+    [ValidateSet("debug", "release")]
+    [string]$Configuration = 'debug',
+    [int]$BuildNumber,
+    [switch]$SkipRestore,
+    [switch]$CleanCache,
+    [string]$SimpleVersion = '1.0.0',
+    [string]$SemanticVersion = '1.0.0-zlocal',
+    [string]$PackageSuffix,
+    [string]$Branch,
+    [string]$CommitSHA,
+    [string]$BuildBranch = '5fd8377a9abf3ff411918dbb973948a6677432db'
```

Comment:
I would use a more recent version of the build scripts, for example 1f8e102c9f36a3f06b23fb22cae76d655b033362, especially given that @joelverhagen introduced a `New-WebAppPackage` command that I think it's highly likely this build will eventually use.